packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/__tests__/Agreement.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { renderForm } from "../../../../../test/renderForm";
+import { AgreementStep } from "../AgreementStep";
+
+describe("<AgreementStep />", () => {
+  it("should render", () => {
+    const { getAllByRole, queryByText } = renderForm(AgreementStep);
+
+    expect(getAllByRole("radio").length).toBe(2);
+    expect(queryByText(/Je ne souhaite pas renseigner/)).toBeNull();
```

Comment:
Pour simplifier la lecture:

```suggestion
    expect(queryByText(/Je ne souhaite pas renseigner/)).not.toBeInTheDocument();
```